modules/navier_stokes/test/tests/finite_volume/ins/turbulence/lid-driven/lid-driven-turb-energy-wall-physics.i

```diff
@@ -148,7 +148,8 @@ wall_treatment_tem = 'eq_linearized' # Options: eq_newton, eq_incremental, eq_li
   petsc_options_iname = '-pc_type -pc_factor_shift_type'
   petsc_options_value = 'lu NONZERO'
   nl_rel_tol = 1e-10
-  nl_abs_tol = 1e-10
+  nl_abs_tol = 5e-8
+  nl_max_its = 100
```

Comment:
How many iterations does this thing take?

Reply:
70 to get this low. It stagnates for like 50 iterations before getting better

Reply:
Could you do a jacobian check, just in case?

Reply:
Or try it with, line_search=none, `-snes_lineseach_damping 0.8`

Reply:
there is no way the Jacobian is fully accurate, the BCs and the sink terms use the old state everywhere

Reply:
I was left under the impression from @tanoret 's work into this last year that lagging things was necessary to get convergence in Newton. I did not redo this work

Reply:
I tried to not use the linearized but the steady state is not converging then

Reply:
I added this improvement to the restart PR